pyproject.toml

```diff
@@ -4,11 +4,15 @@ description = "Juju CLI wrapper for charm integration testing"
 readme = "README.md"
 requires-python = ">=3.12"
 dynamic = ["version"]
-
 dependencies = [
     "PyYAML==6.*",
 ]
 
+[project.optional-dependencies]
+pytest = [
```

Comment:
What's the purpose of this optional dependency?

Reply:
Oops, yes, a hangover from when I had `jubilant/fixtures.py` and used pytest there. No longer needed -- removed.